layouts/_shortcodes/asciinema.html

```diff
@@ -0,0 +1,48 @@
+{{- /* Get parameters */ -}}
+{{- $castFile := .Get "file" | default (.Get 0) -}}
+{{- /* Handle file path: if not absolute path or URL, prepend /casts/ */ -}}
+{{- if and (not (hasPrefix $castFile "/")) (not (hasPrefix $castFile "http://")) (not (hasPrefix $castFile "https://")) -}}
+  {{- $castFile = printf "/casts/%s" $castFile -}}
+{{- end -}}
```

Comment:
I noticed that it prepends `/casts/` to paths, which might be confusing to users, so I recommend we raise an error instead of attempting to force the location.